mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown

```diff
@@ -67,11 +67,15 @@ If you are using Terraform on your workstation we recommend that you install
 as a primary authentication method. You can enable ADCs by running the command
 `gcloud auth application-default login`.
 
+<!-- 
+TODO: quota project is not currently read from ADC file b/360405077#comment8
```

Comment:
Just to confirm: this also means that the `gcloud` commands below don't work, right? Since that ultimately works through a local credentials file?

So removing this statement in our docs reflects current behaviour.

Reply:
This file is [specifically read](https://github.com/golang/oauth2/blob/22134a41033e44c2cd074106770ab5b7ca910d15/google/default.go#L304):
`$HOME/.config/gcloud/application_default_credentials.json`

We [preserve only](https://github.com/hashicorp/terraform-provider-google/blob/6b679462b2ee101a9a903c8f4370d40d8aebd8aa/google/transport/config.go#L1578-L1586) the [token source](https://github.com/hashicorp/terraform-provider-google/blob/6b679462b2ee101a9a903c8f4370d40d8aebd8aa/google/transport/config.go#L2205).

Reply:
Thanks! Does that mean that quota projects _are_ read from the other ADC sources?

Reply:
nope quota project is never read as far as I can tell. We always throw out everything in favor of the token source, which we pass to transport library. The token source doesn't contain any special information